scripts/cc-cli/src/test/parsing.test.ts

```diff
@@ -0,0 +1,146 @@
+import {
+  parseAddress,
+  parseAmount,
+  parseBoolean,
+  parseChoice,
+  parseHexString,
+  parseInteger,
+  parsePercentAsPerbill,
+} from "../utils/parsing";
+
+describe("parseAddress", () => {
+  test("parsed valid address returns same address", () => {
+    const substrateAddress = "5EACfEfYjfg5ZHpzp9uoMCR68UNGBUAu5AYjvZdM5aLYaojx";
+    const parsedAddress = parseAddress(substrateAddress);
+    expect(parsedAddress).toBe(substrateAddress);
+  });
+
+  test("parsed invalid address throws error", () => {
+    const substrateAddress = "5EACfEfYjfg5ZHpzp9uoMCZdM5aLYaojx";
+    const parsedInvalid = () => parseAddress(substrateAddress);
+    expect(parsedInvalid).toThrowError(Error);
+  });
+});
+
+describe("parseAmount", () => {
+  test("parsed valid amount returns same amount * 10^^18", () => {
+    const amount = "100";
+    const parsedAmount = parseAmount(amount);
+    expect(parsedAmount.toString()).toBe("100000000000000000000");
+  });
+
+  test("parsed negative amount throws error", () => {
+    const amount = "-100";
+    const parsedInvalid = () => parseAmount(amount);
+    expect(parsedInvalid).toThrowError(Error);
+  });
+
+  test("parsed invalid decimal char throws error", () => {
+    const amount = "100,1";
```

Comment:
Decimal comma is a valid specifier in some countries. IDK if we want to specially handle this or not.

Reply:
Used this one for kinda documenting what `ethers` is doing. Supporting other separators means going on to customize parsing.

What do you think about adding a warning to inform those who try using a comma?

Reply:
I think we need to discuss this first and make a decision afterwards. Opened CSUB-645 for that.